src/applications/find-forms/tests/e2e/00-required.cypress.spec.js

```diff
@@ -15,12 +18,50 @@ function axeTestPage() {
   cy.axeCheck();
 }
 
+const sortTheResults = (indexA, indexB) => {
```

Comment:
Looks like this method is copied over the from the class, could we abstract it to only use one `sortTheResults` that's shared between the 2? That way they don't get out of sync on accident in the future

Reply:
We definitely can.